flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/source/IcebergSource.java

```diff
@@ -349,6 +401,8 @@ public IcebergSource<T> build() {
         throw new UncheckedIOException(e);
       }
 
+      contextBuilder.fromProperties(table, readOptions, flinkConfig);
```

Comment:
`contextBuilder` doesn't need to be a class member anymore. can be a local/stack variable in this method.

Reply:
The `filters` and `project` still use it.